android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonsProvider.kt

```diff
@@ -10,17 +10,17 @@ package mozilla.components.feature.addons
 interface AddonsProvider {
 
     /**
-     * Provides a list of all available add-ons.
+     * Provides a list of all featured add-ons.
```

Comment:
For consumers using A/C. Should we expand on what `feature` means?

Reply:
Good call. Not sure how to explain it, though. For us (or rather Fenix), this is about having a list of recommended/featured/curated add-ons to list in the Fenix add-ons manager...

Reply:
👍🏽